src/core/RecyclerListView.tsx

```diff
@@ -108,6 +115,12 @@ export interface RecyclerListViewProps {
     //For all props that need to be proxied to inner/external scrollview. Put them in an object and they'll be spread
     //and passed down. For better typescript support.
     scrollViewProps?: object;
+
+    /**
+     * Use this prop to provide padding around list content instead of
+     * using padding/margin in contentContainerStyle prop of scroll view.
+     */
+    contentInsets?: EdgeInsets;
```

Comment:
Won't this conflict with existing ScrollView prop?

Reply:
Can we just take paddingTop etc directly which we can override in give contentContainerStyle.
Also will need to be handled in web version